jobs/integration/validation.py

```diff
@@ -744,11 +744,17 @@ async def test_worker_master_removal(model, tools):
 
 
 @pytest.mark.on_model("validate-nvidia")
-@pytest.mark.skip("Feature removed in ops rewrite")
-async def test_gpu_support(model, tools):
+async def test_gpu_support(model, k8s_model, tools):
     """Test gpu support. Should be disabled if hardware
     is not detected and functional if hardware is fine"""
 
+    # Deploy nvidia-gpu-operator charm
+    # Trust is True because the charm needs privileges to install drivers
+    # and packages on the workers
+    await k8s_model.deploy(
+        entity_url="nvidia-gpu-operator", channel="1.29/stable", trust=True
+    )
```

Comment:
Stick to stable so it's not nailed into one release.

```suggestion
    await k8s_model.deploy(
        entity_url="nvidia-gpu-operator", channel="latest/stable", trust=True
    )
```